src/platform/forms/save-in-progress/ApplicationStatus.jsx

```diff
@@ -1,16 +1,15 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import moment from 'moment';
-import { connect } from 'react-redux';
-
+import LoadingIndicator from '@department-of-veterans-affairs/formation-react/LoadingIndicator';
+import Modal from '@department-of-veterans-affairs/formation-react/Modal';
+import ProgressButton from '@department-of-veterans-affairs/formation-react/ProgressButton';
 import {
-  formLinks,
-  formDescriptions,
   formBenefits,
+  formDescriptions,
+  formLinks,
 } from 'applications/personalization/dashboard/helpers';
-import LoadingIndicator from '@department-of-veterans-affairs/formation-react/LoadingIndicator';
-import ProgressButton from '@department-of-veterans-affairs/formation-react/ProgressButton';
-import Modal from '@department-of-veterans-affairs/formation-react/Modal';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
 import { removeSavedForm } from '../../user/profile/actions';
```

Comment:
More import reordering noise...

Reply:
yeah, I got confused when I saw `react` removed

Reply:
Will have to disable my ESLint plugin to avoid the confusion.

Reply:
You shouldn't have to disable ESLint- that's the only recommended plugin. I think there's a config or rule that's not part of the platform that's causing this.

Reply:
@rianfowler Let me know if you find out which rule/config it is.

Reply:
What IDE are you using and what are the plugins?